module/Database/view/database/index/index.phtml

```diff
@@ -11,9 +11,13 @@ use Laminas\View\Renderer\PhpRenderer;
  * @var int $expired
  * @var int $prospectives
  * @var int $updates
+ * @var bool $syncPaused
+ * @var DateTime $syncPausedUntil
+ * @var int $totalCount
  */
 
-$showPendingUpdates = 0 < $updates;
+$twoColumnMode = $totalCount > 0;
+$syncPausedDifference = $syncPausedUntil->getTimestamp() - (new DateTime())->getTimestamp();
```

Comment:
`syncPausedUntil` can be `null` (if the specific config item is not yet configured) causing this to throw an error.